src/models/pixelart_diffusion.py

```
@@ -73,18 +77,29 @@ def default_diffusion_config() -> PixelArtDiffusionConfig:
         post_grid=post_grid,
         post_palette=post_palette,
+        lora_repo=lora_repo or None,
+        lora_weight_name=lora_weight_name or None,
     )
 
 
 class PixelArtDiffusionStylizer:
     def __init__(self, config: PixelArtDiffusionConfig):
-        if not os.path.exists(config.lora_path):
+        if not os.path.exists(config.lora_path) and not config.lora_repo:
             raise FileNotFoundError(f"pixel-art LoRA not found: {config.lora_path}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)